lib/wallaby/browser.ex

```diff
@@ -857,6 +857,17 @@ defmodule Wallaby.Browser do
     end
   end
 
+  defp do_get(query, elements) do
+    case {Query.get_number(query), length(elements)} do
+      {nil,_} ->
+        {:ok, elements}
+      {n, count} when n > 0 and n <= count ->
```

Comment:
I'm not sure how I feel about starting at 1 here. The programmer in me wants to 0-index everything.